README.md

```diff
@@ -1,89 +1,96 @@
 <div align="center">
-	<h1>2Based2Wait</h1>
+	<h2>2Based2Wait</h2>
 	<p>Lightweight & (theoretically) extensible 2b2t proxy.</p>
-	<h4>
-		<a href="https://github.com/Enchoseon/2based2wait/wiki">Wiki</a>
-		<span> · </span>
-		<a href="https://github.com/Enchoseon/2based2wait/issues">Report Bug</a>
-		<span> · </span>
-		<a href="https://github.com/Enchoseon/2based2wait/blob/main/docs/configuration-guide.md">Configuration Guide</a>
-	</h4>
+	<strong>
+	<a href="https://github.com/Enchoseon/2based2wait/wiki">Wiki</a>
+	<span> · </span>
+	<a href="https://github.com/Enchoseon/2based2wait/issues/new">Report a Bug</a>
+	<span> · </span>
+	<a href="https://github.com/Enchoseon/2based2wait/blob/main/docs/configuration-guide.md">Configuration Guide</a>
+	<span> · </span>
+	<a href="https://github.com/Enchoseon/2based2wait/discussions">Discussions</a>
+	</strong>
+	<br><br>
 	<img src="https://img.shields.io/github/last-commit/Enchoseon/2based2wait?color=2A0944&labelColor=525E75&style=flat" alt="Last Commit">
 	<img src="https://img.shields.io/github/languages/code-size/Enchoseon/2based2wait?color=3FA796&labelColor=525E75&style=flat" alt="Code Size">
 	<img src="https://img.shields.io/github/package-json/v/Enchoseon/2based2wait?color=FEC260&labelColor=525E75&style=flat" alt="Current Version">
 	<img src="https://img.shields.io/github/license/Enchoseon/2based2wait?color=A10035&labelColor=525E75&style=flat" alt="License">
 </div>
 
-<br />
+## Table of Contents
 
-# Table of Contents
-
-- [Quick Start](#quick-start)
+- [Installation](#installation)
 - [Features](#features)
 - [Images](#images)
 - [For Developers](#for-developers)
-- [Keep in Mind](#keep-in-mind)
 
-# Quick Start
-- This project utilizes pnpm (performant npm) as its package manager for improved performance. If you don't have pnpm installed, you can use npm to install it with the following command. :
- `npm install -g pnpm `
+## Installation
+
+### Prerequisites
+
+- [Git](https://git-scm.com/)
+- [NodeJS](https://nodejs.org/)
+- [pnpm](https://pnpm.io/): `npm install -g pnpm`
+
+### Quick Start
 
-1. In `config.json`, configure the following values:
-    - `account.username`: Your Minecraft account playername.
-    - `proxy.whitelist`: Playernames of accounts that are allowed to connect to the proxy.
-2. Run `pnpm i --prod`
-3. Run `pnpm start`
-4. Enter the auth code that appears in the console into microsoft.com/link
-5. Connect to `127.0.0.1` in Minecraft
-    - If `127.0.0.1` doesn't work try `0.0.0.0`
+1. Clone the repository: `git clone https://github.com/Enchoseon/2based2wait` and `cd 2based2wait`
```

Comment:
```suggestion
1. Clone the current stable release: `git clone git@github.com:Enchoseon/2based2wait.git --branch v1.0.4` and `cd 2based2wait`
```
Tell users to clone latest stable rather than main.